joomla/joomla-cms
# Add user note edit view for Hathor

This PR solves issue #7836. It will add a template override to the Hathor template for the com_users/note/edit.php
## How to test

1) Set Hathor as default admin template
2) Go to Users -> User Notes and press the new button
3) Before applying the patch you will see this:
![687474703a2f2f6973737565732e6a6f6f6d6c612e6f72672f75706c6f6164732f312f64383939326132313762383265303263333535633332383636363065373362612e706e67](https://cloud.githubusercontent.com/assets/2659866/10418557/74dfbf68-705e-11e5-93a8-3396abaf117f.png)

After applying the patch:
![knipsel](https://cloud.githubusercontent.com/assets/2659866/10418559/918ad418-705e-11e5-9cd3-e9cd9436d398.PNG)

4) Also confirm the layout is the same for editing an user note
